libmamba/include/mamba/core/solver.hpp

```diff
@@ -95,7 +95,7 @@ namespace mamba
         [[nodiscard]] const std::vector<MatchSpec>& neuter_specs() const;
         [[nodiscard]] const std::vector<MatchSpec>& pinned_specs() const;
 
-
+        operator const Solver*() const;
```

Comment:
I would prefer the conversions to be explicit, maybe better provided by a function like `raw()` that we used somewhere else?

Reply:
Not sure why this appears in the diff but this is current behavior. The isolation of libsolv into `mamba/solv-cpp` will surely get rid of these conversion, but this is not the scope of this PR.

Reply:
Indeed these implicit conversions are everywhere regarding libsolv objects, and will be removed progressively while they are isolated from libmamba.